src/engine/Drawing/Sprite.ts

```diff
@@ -1,5 +1,7 @@
 module ex {
 
+   var clamp = ex.Util.clamp;
```

Comment:
Sure, but I mean do we **really** need it? We're already in the `ex` module, why not just do `Util.clamp`. I know you didn't do it, but while we're here maybe we should just clean it up ;)